front-end/packages/robustar/src/components/image-editor/ImageEditor.vue

```diff
@@ -97,23 +98,25 @@ export default {
       let options = editorDefaultOptions;
       if (this.includeUi) {
         options = Object.assign(includeUIOptions, this.options);
-        options.includeUI.loadImage = this.image;
+        const timestamp = new Date().getTime();
+        const path = `${configs.imagePathServerUrl}?image_url=${this.base64}&timestamp=${timestamp}`;
+        const name = this.url;
+        options.includeUI.loadImage = {
+          path: path,
+          name: name,
+        };
       }
       this.editorInstance = new ImageEditor('.tui-image-editor', options);
       if (!this.includeUi) {
-        const { path, name } = this.image;
-        this.editorInstance
-          .loadImageFromURL(path, name) 
-          .then(() => {
-            this.editorInstance.clearUndoStack();
```

Comment:
I can see in the updated code that `clearUndoStack( )` is no longer called. Looks like everything still works as expected as I tested it, but I'm still not entirely sure if removing this function call matters. Could you also double check just to make sure? 

cc @shuguang-lv